client/components/session-stats.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import getDisplayTime from '../lib/get-display-time';
+
+export default class SessionStats extends React.Component {
+
+  getBest(times) {
+    let best;
+    times.length > 0 ? best = getDisplayTime(Math.min(...times)) : best = 'N/A';
+    return best;
+  }
+
+  getWorst(times) {
+    let worst;
+    times.length > 0 ? worst = getDisplayTime(Math.max(...times)) : worst = 'N/A';
+    return worst;
+  }
+
+  getAverage(times) {
+    let avg;
+    times.length > 0 ? avg = getDisplayTime(times.reduce((acc, cur) => acc + cur) / times.length) : avg = 'N/A';
+    return avg;
+  }
+
+  getMedian(times) {
+    const timesCopy = times.slice();
+    let median;
+    if (times.length > 0) {
+      timesCopy.sort((a, b) => a - b);
+      const split = Math.floor(timesCopy.length / 2);
+      if (timesCopy.length % 2) {
+        median = getDisplayTime(timesCopy[split]);
+      } else {
+        median = getDisplayTime((timesCopy[split] + timesCopy[split - 1]) / 2);
+      }
+    } else {
+      median = 'N/A';
+    }
+    return median;
```

Comment:
This code can be simplified if you try to get your special case of `0` length out of the way immediately.
```js
if (time.length === 0) return 'N/A';
// calculate and return median
```